src/server/README.md

```diff
@@ -1030,3 +1000,33 @@ Messaging API
                     }[]
             -   errors:
                 -   Internal server error -> Status code: 500
+
+    -   api/message/get-by-email/:email
+        -   HTTP Method: GET
+        -   Get a user object given an email of the user.
```

Comment:
Gets

Reply:
I don't think this is the description for the message/get-by-email API. It seems like this is the description for the user/authenticated API.

Reply:
Oh sorry, I should delete the "isAuthenticated: boolean," line.

Reply:
If the rest of the response matches what you return then yeah. Also, I don't think you need to put the Cookies section either

Reply:
Done.